src/transformState.js

```diff
@@ -1,11 +1,29 @@
 'use strict';
 
-/**
- * @param {Object} state
- * @param {Object[]} actions
- */
 function transformState(state, actions) {
-  // write code here
+  const modifiedState = state;
```

Comment:
Not necessary to create a separate variable, since in the case of objects you assign reference, so by changing `modifiedState` you change  `state` as well